packages/angular/src/lib/legacy/router/page-router-outlet.ts

```diff
@@ -162,7 +203,7 @@ export class PageRouterOutlet implements OnDestroy, RouterOutletContract {
     private router: Router,
     elRef: ElementRef,
     viewUtil: ViewUtil,
-    private environmentInjector: EnvironmentInjector
+    private environmentInjector: EnvironmentInjector,
```

Comment:
It's not by an means a show stopper, but the , character has been added in a few places where it wasn't needed. It won't prevent the code from compiling, therefore it isn't an issue.